__tests__/app.test.js

```diff
@@ -13,10 +15,9 @@ afterAll(() => db.end());
             .get('/api/topics')
             .expect(200)
             .then(({body}) => {
-                console.log(body, '<<< test block received');
-                expect(body).toBeInstanceOf(Array) //can I delete this if a forEach works successfully, as that is proof it is an array ?
-                body.forEach((topic) => {
-                    expect(topic).toEqual(
+                const topics = body.topics
+                topics.forEach((element) => {
```

Comment:
don't forget to make sure that any arrays you're iterating over in a test aren't empty (ie. checking the length), otherwise an empty array will give you a false positive on this test

Reply:
resolved :)